package.json

```diff
@@ -14,23 +14,23 @@
     "test": "npm-run-all --parallel test:unit test:unit:monorepo test:testcafe"
   },
   "files": [
-    "dist",
-    "typings"
+    "dist"
   ],
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@babel/runtime": "^7.4.3",
-    "@testing-library/dom": "^7.0.2",
-    "testcafe": "^1.5.0"
+    "@testing-library/dom": "^7.2.2",
+    "testcafe": "^1.8.4"
```

Comment:
pull latest master after #147 is merged. testcafe should be a peer dep

Reply:
Done!